nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/access-policies/ui/global-access-policies/global-access-policies-routing.module.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { GlobalAccessPolicies } from './global-access-policies.component';
+import { authorizationGuard } from '../../../../service/guard/authorization.guard';
+import { CurrentUser } from '../../../../state/current-user';
+
+const routes: Routes = [
+    {
+        path: ':action/:resource',
+        canMatch: [
+            authorizationGuard(
+                (user: CurrentUser) =>
+                    user.tenantsPermissions.canRead &&
+                    user.policiesPermissions.canRead &&
+                    user.policiesPermissions.canWrite
+            )
```

Comment:
Do we also need to inject the FlowConfiguration here to verify `supportsManagedAuthorizer`?
If the user has the url, http://localhost:4200/#/access-policies/global/read/parameter-contexts for example, they can get to the page even though the menu options are not available.

Same is true for the users page as well.